tests/test_utils.py

```diff
@@ -139,15 +140,17 @@ def check_if_job_finished(job: JobDataclass):
     assert job.state == JobState.FINISHED
 
 
-def ibm_check_if_job_runner_result_correct(job: JobDataclass):
-    job.type = JobType.RUNNER
+def check_if_job_runner_result_correct(job: JobDataclass):
+    """Iterate over every result and check if the distribution of the measurement is correct"""
     for i in range(len(job.results)):
         result: ResultDataclass = job.results[i]
         check_standard_result_data(i, job, result)
         assert result.meta_data is not None
         shots: int = job.shots
         counts: dict = result.result_dict["counts"]
         probabilities: dict = result.result_dict["probabilities"]
+
+        # Check if the first result is correct distributed 50% for the qubit zero and three
```

Comment:
'Check if the first result is distributed correctly: 50% for the qubit zero and 50% for the qubit three"